openmc/model/surface_composite.py

```diff
@@ -783,6 +784,41 @@ def _make_ccw(self, points):
 
         return points[::-1, :]
 
+    def _constrain_triangulation(self, points):
```

Comment:
What do you think about adding a test for this function given the points from the reproducer?